docs/plus/index.md

```diff
@@ -24,6 +24,42 @@ any spec version higher or equal to `1.22.0`. **Non-stable** resources are not g
 
     If you are deploying manifests produced by `cdk8s-plus-22` onto clusters of a lower version, you might encounter some unsupported spec properties or invalid manifests.
 
+## FAQ
+
+### What's the difference between `cdk8s-plus-20`, `cdk8s-plus-21`, and `cdk8s-plus-22`?
+
+These are separately vended libraries that each target a different kubernetes
+version, marked by the `-XX` suffix. For example, `cdk8s-plus-22` targets
+kubernetes version `1.22.0`.
+
+We offer a dedicated version per Kubernetes version to allow users to match
+their manifests to the Kubernetes version they are operating. This way, users
+are only exposed to a set of capabilities offered by their specific cluster,
+preventing deployment errors caused by version mismatches.
+
+### I operate Kubernetes version `1.XX` - which cdk8s+ library should I be using?
+
+If there is a `cdk8s-plus-XX` library that matches your target Kubernetes
+version, we recommend using it since all Kubernetes manifests generated using it
+will be compatible.
+
+If there is not a matching `cdk8s-plus-XX` library, we recommend using the
+closest matching version. The manifests generated by `cdk8s-plus-XX` may also
+work for older versions of Kubernetes, but you might encounter some unsupported
+spec properties or invalid manifests.
+
+### I'm using `cdk8s-plus-XX` - which kubernetes versions will my manifest work on?
+
+If you are using stable APIs (those that are not in alpha or beta), manifests
+generated in `cdk8s-plus-XX` will work in Kubernetes versions `1.XX.0` and
+above. Unstable APIs (which are always labeled in cdk8s+ using a suffix, e.g.
+`IngressV1Beta1`) may work in newer versions of Kubernetes, but it is also
+possible they have been removed.
+
+The manifests generated by `cdk8s-plus-XX` may also work for older versions of
+Kubernetes, but you might encounter some unsupported spec properties or invalid
+manifests.
```

Comment:
This sentence is repeated from the section above, but I think that is OK. I'm mostly aiming for brevity and accuracy

Reply:
👍